core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -589,10 +589,11 @@ private[spark] class BlockManager(
   private def doGetRemote(blockId: BlockId, asBlockResult: Boolean): Option[Any] = {
     require(blockId != null, "BlockId is null")
     val locations = Random.shuffle(master.getLocations(blockId))
+    val timeout = conf.getInt("spark.storage.fetchBlockTimeout", 1000) // seconds
```

Comment:
This code was written a while ago, but now we actually use the `1000s` notation instead of assuming a particular unit with time configs. We'll need to update that here.